tutorials/Quantile_Interval_Score.ipynb

```diff
@@ -0,0 +1,1384 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantile Interval Score\n",
+    "\n",
+    "The quantile interval score is a consistent scoring function to evaluate prediction intervals, where the aim is to predict not just a single point value, but also an interval that captures uncertainty at a given confidence level.\n",
+    "\n",
+    "The quantile interval score function is defined as:\n",
+    "\n",
+    "$$ S(q_l, q_u, y) = (q_{u} - q_{l}) +\n",
+    "\\frac{1}{\\alpha_l} \\cdot (q_{l} - y) \\cdot \\mathbb{1}(y < q_{l}) + \\frac{1}{1 - \\alpha_u} \\cdot (q_{u} - y) \\cdot \\mathbb{1}(y < q_{u}) $$\n",
+    "\n",
+    "where $S$ is the scoring function (here *quantile interval score*), $q_l$ and $q_u$ are lower and upper quantile forecasts, and $y$ is observation, $\\alpha_l$ and $\\alpha_u$ are lower and upper quantile level. $\\mathbb{1}(condition)$ is an indicator function that is 1 when the condition is true, and 0 otherwise. Lower values of the quantile interval scores are better. As you can see, this score penalizes the width of interval as well as whether observation is outside of the interval forecast (i.e., under-prediction penalty when observation is larger than upper-quantile forecast, and over-prediction penalty when observation is smaller than lower-quantile forecast).\n",
+    "\n",
+    "Please note that *quantile interval score* can accept any values for lower and upper quantile level as long as $0 < \\alpha_l < \\alpha_u < 1$ is met. So *quantile interval score* is equivalent to the *interval score*, however it does not have the constraint of *interval score*, where quantile level range has to be symmetric (i.e., for example 0.2 and 0.9, lower and upper quantile levels, respectively, cannot be use for *interval score*).\n",
+    "\n",
+    "To learn more about *quantile interval score* please see the following article:  \n",
+    "\n",
+    "- T. Gneiting, and Raftery A.E., \"Strictly proper scoring rules, prediction and estimation\", J. Amer. Stat. Assoc., Vol. 102 No. 477 (2007), pp. 359--378, Corollary 5.2. [https://doi.org/10.1198/016214506000001437](https://doi.org/10.1198/016214506000001437)\n",
```

Comment:
For references, we have been moving towards using the APA 7 style guide, see: https://apastyle.apa.org/style-grammar-guidelines/references/examples/journal-article-references

Reply:
It has now been fixed